libsigner/src/messages.rs

```diff
@@ -233,6 +386,46 @@ impl StacksMessageCodec for SignerMessage {
                 let transactions = read_next::<Vec<StacksTransaction>, _>(fd)?;
                 SignerMessage::Transactions(transactions)
             }
+            SignerMessageTypePrefix::DkgResults => {
+                let aggregate_key = Self::deserialize_point(fd)?;
+                let party_polynomial_len = u32::consensus_deserialize(fd)?;
+                let mut party_polynomials = Vec::with_capacity(
+                    party_polynomial_len
+                        .try_into()
+                        .expect("FATAL: u32 could not fit in usize"),
+                );
+                for _ in 0..party_polynomial_len {
+                    let party_id = u32::consensus_deserialize(fd)?;
+                    let polynomial_id_id = Self::deserialize_scalar(fd)?;
+                    let polynomial_id_kg = Self::deserialize_point(fd)?;
+                    let polynomial_id_kca = Self::deserialize_scalar(fd)?;
+
+                    let commit_len = u32::consensus_deserialize(fd)?;
+                    let mut polynomial_poly = Vec::with_capacity(
+                        commit_len
+                            .try_into()
+                            .expect("FATAL: u32 could not fit in usize"),
+                    );
+                    for _ in 0..commit_len {
+                        let poly = Self::deserialize_point(fd)?;
+                        polynomial_poly.push(poly);
+                    }
+                    let polynomial_id = ID {
+                        id: polynomial_id_id,
+                        kG: polynomial_id_kg,
+                        kca: polynomial_id_kca,
+                    };
+                    let polynomial = PolyCommitment {
```

Comment:
If we're maintaining a fork of `wsts` anyway, should we instead create a `StacksMessageCodec` implementation for `PolyCommitment` over there instead?

Reply:
Then it would need to depend on `stacks-common`-- I think it's better to just have a serialization specific to the DKG results message.